includes/data-stores/class-wc-order-data-store-cpt.php

```diff
@@ -76,7 +76,6 @@ class WC_Order_Data_Store_CPT extends Abstract_WC_Order_Data_Store_CPT implement
 	public function create( &$order ) {
 		$order->set_order_key( 'wc_' . apply_filters( 'woocommerce_generate_order_key', uniqid( 'order_' ) ) );
 		parent::create( $order );
-		do_action( 'woocommerce_new_order', $order->get_id() );
```

Comment:
I'm not sure I agree totally there. If the `woocommerce_new_order` hook passed an cart object before and we removed that, that would be a breaking change. This is not the case though - woocommerce_new_order still fires in the same way, just in additional places like API.

@thenbrent any thoughts on this one?